mainwindow.ui

```diff
@@ -52,7 +55,7 @@
             <string>Launch</string>
            </property>
            <property name="icon">
-            <iconset resource="qtresources.qrc">
+            <iconset theme="system-run" resource="qtresources.qrc">
```

Comment:
I think this is actually a more general complaint about icon themes. I want to make the application look like I want, not to specify vague concepts and hope that the icons installed in those slots happen to look good.

For instance, I've picked a green right-pointing arrow for this. I'm sure there's a generic 'right-arrow' icon slot, but there's no guarantee that the icon for it will be green. Likewise, the white 'home' icon fits nicely alongside the white-page notebook icons:

![screenshot from 2015-07-15 17 03 52](https://cloud.githubusercontent.com/assets/327925/8712633/9d94fc42-2b13-11e5-8108-4082fadaa9f2.png)

When that gets replaced by an arbitrary 'home' icon, it doesn't look so good:

![screenshot from 2015-07-15 17 04 23](https://cloud.githubusercontent.com/assets/327925/8712641/ac41a29a-2b13-11e5-9f27-1af3de5d23e1.png)